src/components/scene/reflection.js

```diff
@@ -142,7 +143,7 @@ module.exports.Component = register('reflection', {
       this.needsVREnvironmentUpdate = false;
       this.cubeCamera.position.set(0, 1.6, 0);
       this.cubeCamera.update(renderer, scene);
-      this.el.object3D.environment = this.cubeRenderTarget.texture;
+      scene.environment = this.cubeRenderTarget.texture;
```

Comment:
We use `scene.environment = null` three line above, we should use the scene variable here for symmetry :)